src/input.rs

```diff
@@ -556,10 +538,14 @@ impl<'a, A: ActionContext + 'a> Processor<'a, A> {
         let launcher = self.mouse_config.url.launcher.as_ref()?;
         let mut args = launcher.args().to_vec();
         args.push(text);
-
-        match Command::new(launcher.program()).args(&args).spawn() {
-            Ok(_) => debug!("Launched: {} {:?}", launcher.program(), args),
-            Err(_) => warn!("Unable to launch: {} {:?}", launcher.program(), args),
+        let spawned=start_daemon(launcher.program(),&args);
+        match spawned {
+            Ok(child) => {
+                debug!("Launched: {} {:?}", launcher.program(), args);
+            },
+            Err(err) => {
+                warn!("Unable to launch: {} {:?}", launcher.program(), args);
+            },
```

Comment:
The variables `child` and `err` are not used here and should not be defined.

These lines should stay unchanged since it's not necessary to alter anything here.

Reply:
These should be changed back to single-line, the way they have been before this PR.